core/math/color_names.inc

```diff
@@ -13,151 +13,151 @@ struct NamedColor {
-	{ "SANDY_BROWN", Color(0.96, 0.64, 0.38) },
-	{ "SEA_GREEN", Color(0.18, 0.55, 0.34) },
-	{ "SEASHELL", Color(1.00, 0.96, 0.93) },
-	{ "SIENNA", Color(0.63, 0.32, 0.18) },
-	{ "SILVER", Color(0.75, 0.75, 0.75) },
-	{ "SKY_BLUE", Color(0.53, 0.81, 0.92) },
-	{ "SLATE_BLUE", Color(0.42, 0.35, 0.80) },
-	{ "SLATE_GRAY", Color(0.44, 0.50, 0.56) },
-	{ "SNOW", Color(1.00, 0.98, 0.98) },
-	{ "SPRING_GREEN", Color(0.00, 1.00, 0.50) },
-	{ "STEEL_BLUE", Color(0.27, 0.51, 0.71) },
-	{ "TAN", Color(0.82, 0.71, 0.55) },
-	{ "TEAL", Color(0.00, 0.50, 0.50) },
-	{ "THISTLE", Color(0.85, 0.75, 0.85) },
-	{ "TOMATO", Color(1.00, 0.39, 0.28) },
-	{ "TRANSPARENT", Color(1.00, 1.00, 1.00, 0.00) },
-	{ "TURQUOISE", Color(0.25, 0.88, 0.82) },
-	{ "VIOLET", Color(0.93, 0.51, 0.93) },
+	{ "ALICE_BLUE", Color::hex(0xF0F8FFFF) },
+	{ "ANTIQUE_WHITE", Color::hex(0xFAEBD7FF) },
+	{ "AQUA", Color::hex(0x00FFFFFF) },
+	{ "AQUAMARINE", Color::hex(0x7FFFD4FF) },
+	{ "AZURE", Color::hex(0xF0FFFFFF) },
+	{ "BEIGE", Color::hex(0xF5F5DCFF) },
+	{ "BISQUE", Color::hex(0xFFE4C4FF) },
+	{ "BLACK", Color::hex(0x000000FF) },
+	{ "BLANCHED_ALMOND", Color::hex(0xFFEBCDFF) },
+	{ "BLUE", Color::hex(0x0000FFFF) },
+	{ "BLUE_VIOLET", Color::hex(0x8A2BE2FF) },
+	{ "BROWN", Color::hex(0xA52A2AFF) },
+	{ "BURLYWOOD", Color::hex(0xDEB887FF) },
+	{ "CADET_BLUE", Color::hex(0x5F9EA0FF) },
+	{ "CHARTREUSE", Color::hex(0x7FFF00FF) },
+	{ "CHOCOLATE", Color::hex(0xD2691EFF) },
+	{ "CORAL", Color::hex(0xFF7F50FF) },
+	{ "CORNFLOWER_BLUE", Color::hex(0x6495EDFF) },
+	{ "CORNSILK", Color::hex(0xFFF8DCFF) },
+	{ "CRIMSON", Color::hex(0xDC143CFF) },
+	{ "CYAN", Color::hex(0x00FFFFFF) },
+	{ "DARK_BLUE", Color::hex(0x00008BFF) },
+	{ "DARK_CYAN", Color::hex(0x008B8BFF) },
+	{ "DARK_GOLDENROD", Color::hex(0xB8860BFF) },
+	{ "DARK_GRAY", Color::hex(0xA9A9A9FF) },
+	{ "DARK_GREEN", Color::hex(0x006400FF) },
+	{ "DARK_KHAKI", Color::hex(0xBDB76BFF) },
+	{ "DARK_MAGENTA", Color::hex(0x8B008BFF) },
+	{ "DARK_OLIVE_GREEN", Color::hex(0x556B2FFF) },
+	{ "DARK_ORANGE", Color::hex(0xFF8C00FF) },
+	{ "DARK_ORCHID", Color::hex(0x9932CCFF) },
+	{ "DARK_RED", Color::hex(0x8B0000FF) },
+	{ "DARK_SALMON", Color::hex(0xE9967AFF) },
+	{ "DARK_SEA_GREEN", Color::hex(0x8FBC8FFF) },
+	{ "DARK_SLATE_BLUE", Color::hex(0x483D8BFF) },
+	{ "DARK_SLATE_GRAY", Color::hex(0x2F4F4FFF) },
+	{ "DARK_TURQUOISE", Color::hex(0x00CED1FF) },
+	{ "DARK_VIOLET", Color::hex(0x9400D3FF) },
+	{ "DEEP_PINK", Color::hex(0xFF1493FF) },
+	{ "DEEP_SKY_BLUE", Color::hex(0x00BFFFFF) },
+	{ "DIM_GRAY", Color::hex(0x696969FF) },
+	{ "DODGER_BLUE", Color::hex(0x1E90FFFF) },
+	{ "FIREBRICK", Color::hex(0xB22222FF) },
+	{ "FLORAL_WHITE", Color::hex(0xFFFAF0FF) },
+	{ "FOREST_GREEN", Color::hex(0x228B22FF) },
+	{ "FUCHSIA", Color::hex(0xFF00FFFF) },
+	{ "GAINSBORO", Color::hex(0xDCDCDCFF) },
+	{ "GHOST_WHITE", Color::hex(0xF8F8FFFF) },
+	{ "GOLD", Color::hex(0xFFD700FF) },
+	{ "GOLDENROD", Color::hex(0xDAA520FF) },
+	{ "GRAY", Color::hex(0x808080FF) },
```

Comment:
Same here, should be `BEBEBEFF`, that's why we have "WEB" variants (these are different between X11 and WEB).